src/CommunityToolkit.Maui/ReadMe.txt

```diff
@@ -24,10 +24,14 @@ In order to make use of the toolkit within XAML you can use this namespace:
 
 xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
 
+!!!WARNING!!!
+In order to use the custom schema mentioned above you need to call the `UseMauiCommunityToolkit` method or call any API on C# code. If you don't do that that url will not work and you need to use the `xmlns= namespace;assembly` pattern. This is a limitation of the custom schema API, more info here:
```

Comment:
could we also add it to this page as well: https://docs.microsoft.com/en-us/dotnet/communitytoolkit/maui/get-started